int32[] ids
string type
string role
string role_type
string scope_id
string scope_role
string scope_role_type
Tag[] tags
---
Node[] nodes
Way[] ways
Relation[] relations
---
